lua/lspconfig/efm.lua

```diff
@@ -7,9 +7,7 @@ local bin_name = "efm-langserver"
 configs[server_name] = {
   default_config = {
     cmd = { bin_name },
-    root_dir = function(fname)
-      return util.root_pattern ".git"(fname) or util.path.dirname(fname)
-    end,
+    root_dir = util.root_pattern ".git",
```

Comment:
This change breaks for me. Using the current directory if outside a git repository makes EFM work for things that are standalone scripts and alike.